ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -141,7 +140,6 @@ export const RuntimePanel = withCurrentWorkspace()(({workspace}) => {
       <div>No runtime exists yet</div>
       {activeRuntimeOp && <hr/>}
```

Comment:
please remove these two lines as well; I noticed the hr/ flashing into existence during the demo